Dockerfile

```diff
@@ -14,17 +14,20 @@ WORKDIR /var/lib/bouncer
 # Copy packaging
 COPY README.rst package.json setup.* ./
 
+# Copy gunivorn config
+COPY gunicorn.conf.py ./
+
+# Copy the rest of the application files
+COPY bouncer ./bouncer/
```

Comment:
The reason this was below `npm install ...` is so that you don't need to rerun the dependency installation unless the dependencies change. By putting it up here you're ensuring that the Docker image cache is busted for any change anywhere in the application. Both this and the `gunicorn.conf.py` should go below the `RUN npm install ...` line.